packages/web-features-figuring/src/baseline/support.ts

```diff
@@ -0,0 +1,23 @@
+import { Browser } from "../browser-compat-data/browser";
+import { Compat } from "../browser-compat-data/compat";
+import { Feature } from "../browser-compat-data/feature";
+import { Release } from "../browser-compat-data/release";
+
+type Support = Map<Browser, Release | undefined>;
+
+export function support(feature: Feature, browsers: Browser[], compat: Compat) {
```

Comment:
Maybe TS infers the type, but as a reader I wasn't so sure. This would help me:

```suggestion
export function support(feature: Feature, browsers: Browser[], compat: Compat): Support {
```